pkg/controller/registry/reconciler/grpc.go

```diff
@@ -198,12 +200,28 @@ func (c *GrpcRegistryReconciler) ensureUpdatePod(source grpcCatalogSourceDecorat
 			updatePod.Labels[CatalogSourceUpdateKey] = ""
 
 			// Update the update pod to promote it to serving pod
-			_, err := c.OpClient.KubernetesInterface().CoreV1().Pods(source.GetNamespace()).Update(context.TODO(), updatePod, metav1.UpdateOptions{})
+			forceApply := true
+			opts := metav1.PatchOptions{
+				Force:        &forceApply,
+				FieldManager: "operator-lifecycle-manager-registry-reconciler",
```

Comment:
nit: we can shorten this down `olm.registry`? and this should either be a member of the `grpcCatalogSourceDecorator` or be defined as a constant.